test/src/e2e_vm_tests/test_programs/should_fail/associated_fn_as_method_call/Forc.toml

```
@@ -0,0 +1,8 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+license = "Apache-2.0"
+name = "associated_fn_as_method_call"
+entry = "main.sw"
+
+[dependencies]
+core = { path = "../../../../../../sway-lib-core" }
```

i think we can omit core here, since we don't use operators in this test. Will help with compile speed.

```suggestion
name = "associated_fn_as_method_call"
entry = "main.sw"
implicit-std = false
```